src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int MAX_RANDOM_VALUE = 100;
+    public static final int DEFAULT_RADIUS = 10;
+    public static final String DEFAULT_COLOR = Colors.White.name();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Figure figure;
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int index = this.random.nextInt(FIGURE_COUNT);
+        switch (index) {
+            case 1:
+                figure = getCircle();
+                break;
+            case 2:
+                figure = getRectangle();
+                break;
+            case 3:
+                figure = getRightTriangle();
+                break;
+            case 4:
+                figure = getIsoscelesTrapezoid();
+                break;
+            case 5:
+                figure = getSquare();
+                break;
+            default:
+                figure = getRightTriangle();
+                break;
+        }
+        figure.setColor(colorSupplier.getRandomColor());
+        return figure;
+    }
+
+    public Figure getCircle() {
```

Comment:
FigureSupplier should have only two public methods(getRandomFigure, getDefaultFigure)

All private methods should be below public methods
```suggestion
    private Figure getCircle() {
```